README.md

```diff
@@ -6,6 +6,70 @@ The HoloToolkit is a collection of scripts and components intended to accelerate
 - Unity Editor Version: [2017.1.0f3](https://unity3d.com/unity/whats-new/unity-2017.1.0)
 - Visual Studio 2017
 
+## Contributing to this project:
+Please go over the [Contributing guidelines](CONTRIBUTING.md) to learn more about the process and thinking.
+
+## Wondering how to simply get started?
+Please go over the [Getting started guide](GettingStarted.md) to learn more about just getting off the ground quickly.
+
+## Release cadence for Master:
+- **Master branch releases** will align with **major Unity releases marking a release every 3 months**. Please read the [Unity roadmap](https://unity3d.com/unity/roadmap).
+- Each release will be marked as a GitHub release tag. You can consume this release by:
+	- Using the zip file available in the release notes
+	- Unity packages available in the release notes
+	- Syncing to the specific git tag you would like to lock on.
+- Release tag name convention: v1.<Unity release major number>.<Unity release minor number>.<Iteration number>
+	- For example: For Unity version 2017.1.0 our release tag would be **v1.2017.1.0**
+	- If we marked another release with the same Unity version: v1.2017.1.1
+- Unity packages for each release will be published to the Unity asset store.
+- Unity packages for each feature area like Input/SpatialMapping will also be added to the release notes.
+- Academy content will be updated with each major toolkit release.
+
+## Roadmap for Master:
+
+| Target Unity release  | Release timeframe | Master release tag | Toolkit release features |
+| --------------------- | ----------------- | ------------------ | ------------------------ |
+| 2017.1.0              | Early Aug 2017    | v1.2017.1.0        | - Updating to Unity's beta build. - 3D Keyboard for text entry.|
+| 2017.2.0              | End Sep 2017      | v1.2017.2.0        | - Updating master with Windows Mixed Reality support (xR namespace). - RI Dev_Unity_2017.2.0 into master. - Toolkit will work on both HoloLens and immersive headsets. - Crystal Key motion controller support.|
+| 2017.3.0              | End Dec 2017      | v1.2017.3.0        | - 360 video player prefabs. - Scriptable foveated rendering prefabs. |
+
+## Release cadence for development branches:
+- Development branches are great for incubating features before they make their way into master.
+- These branches can be feature work or experimental features.
+- Development branches may not be on the same Unity versions as master.
```

Comment:
"might not"

"may not" implies that you cannot be on a different release from master.